src/Calculator/EquationStylePanelControl.xaml.cpp

```diff
@@ -19,13 +20,23 @@ using namespace Windows::UI::Xaml::Input;
 using namespace Windows::UI::Xaml::Media;
 using namespace Windows::UI::Xaml::Navigation;
 using namespace Windows::UI::Xaml::Shapes;
+using namespace GraphControl;
 
 DEPENDENCY_PROPERTY_INITIALIZATION(EquationStylePanelControl, SelectedColor);
+DEPENDENCY_PROPERTY_INITIALIZATION(EquationStylePanelControl, SelectedStyle);
+DEPENDENCY_PROPERTY_INITIALIZATION(EquationStylePanelControl, EnableLineStylePicker);
 DEPENDENCY_PROPERTY_INITIALIZATION(EquationStylePanelControl, AvailableColors);
 
 EquationStylePanelControl::EquationStylePanelControl()
 {
     InitializeComponent();
+
+    auto allStyles = ref new Vector<EquationLineStyle>();
```

Comment:
How may line styles are? I'm seeing 3 here and 5 below. When I tested it, there were only 3.

Reply:
There are only 3 available in the UI, as discussed in the design meeting we only want to use solid, dashed, dotted.